tests/db/test_orm.py

```diff
@@ -0,0 +1,116 @@
+import os
+import unittest
+
+from remoteappmanager.db import orm
+from remoteappmanager.db.orm import Database, transaction, Accounting
+from tests.temp_mixin import TempMixin
+from tests import utils
+
+
+class TestOrm(TempMixin, unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.sqlite_file_path = os.path.join(self.tempdir, "sqlite.db")
+        utils.init_sqlite_db(self.sqlite_file_path)
+
+        self.config = utils.basic_application_config()
+
+    def test_database_init_and_session(self):
+        db = Database(url="sqlite:///"+self.sqlite_file_path)
+        session = db.create_session()
+        self.assertIsNotNone(session)
+
+    def test_orm_objects(self):
+        db = Database(url="sqlite:///"+self.sqlite_file_path)
+        session = db.create_session()
+        with transaction(session):
+
+            users = [orm.User(name="user"+str(i)) for i in range(5)]
+            session.add_all(users)
+
+            teams = [orm.Team(name="team"+str(i)) for i in range(5)]
+            company_team = orm.Team(name='company_team')
+            teams.append(company_team)
+            session.add_all(teams)
+
+            for user, team in zip(users, teams):
+                user.teams.append(team)
+
+            # make users 1 3 and 4 part of the company team
+            users[1].teams.append(company_team)
+            users[3].teams.append(company_team)
+            users[4].teams.append(company_team)
+
+            # verify back population
+            self.assertEqual(teams[0].users[0], users[0])
+            self.assertEqual(len(company_team.users), 3)
```

Comment:
With the intended many-to-many mapping, one should be able to add users 1, 3 and 4 to the company team: `company_team.users.extend([users[1], users[3], users[4]])`
Is this tested too?